templates/CRM/Event/Form/Registration/ThankYou.tpl

```diff
@@ -21,12 +21,10 @@
         </div>
     {/if}
 
-    {* Show link to Tell a Friend (CRM-2153) *}
-    {if $friendText}
-        <div id="tell-a-friend" class="crm-section tell_friend_link-section">
-            <a href="{$friendURL}" title="{$friendText|escape:'html'}" class="button"><span><i class="crm-i fa-chevron-right" aria-hidden="true"></i> {$friendText}</span></a>
-       </div><br /><br />
-    {/if}
+  {* Show link to Tell a Friend (CRM-2153). Other extensions can also add text here but they must purify it *}
+  {foreach from='extensionHtml' item='html'}
```

Comment:
So close!

![untitled3](https://github.com/user-attachments/assets/98fb45d5-d089-4033-8433-5917da3c6739)

Reply:
well as long as your friend's name is 'extension' and their campaign is called 'html' it's spot on - I'd call that a win